lib/utils/constants.dart

```diff
@@ -27,4 +26,34 @@ abstract class Constants {
   static const String cardHealth = "Health: 12";
   static const String cardAbility =
       "Ability: At the end of your turn, add a Dream Card to your hand";
+  static const double iconPadding = 15;
+  static const double subtitleFontSize = 18;
+  static const double titleFontSize = 30;
+  static const String errorMsg = "An error has occurred";
+  static var tileColor = Colors.orange[50];
+  static var cardColor = Colors.orange[100];
```

Comment:
Instead to do this, you can replace it whit Colors.orange.shade50 and Colors.orange.shade100

Reply:
Done